### PR TITLE
Make sure custom alphabet code works properly on Python 2

### PR DESCRIPTION
**Pull request by reuben**
Sorry, I didn't test the code on Python 2 :(

### CODE REVIEW

File: DeepSpeech.py [hunk 1]
```diff
@@ -717,7 +718,8 @@ def calculate_report(results_tuple):
     items = list(zip(*results_tuple))
     mean_wer = 0.0
     for label, decoding, distance, loss in items:
-        corrected = correction(decoding, alphabet)
+        # corrected = correction(decoding, alphabet)
```
Comment (kdavis-mozilla) [inline DeepSpeech.py]: Could you completely remove this? History can keep it for us.

Comment (reuben) [reply DeepSpeech.py]: This was a left over from testing, I undid it. As for removing the correction call/spell.py, I'm leaving that for the PR that integrates the LM into the decoder.

Comment (kdavis-mozilla) [reply DeepSpeech.py]: OK, make sense

**Comment by lissyx** [comment]
r+

**Pull request merged**